test/main.cc

```diff
@@ -11,12 +11,16 @@
 #ifndef BAZEL_BRINGUP
 #include "test/integration/integration.h"
 #endif
+#include "test/test_common/environment.h"
 
 int main(int argc, char** argv) {
   ::testing::InitGoogleMock(&argc, argv);
   Ssl::OpenSsl::initialize();
   Event::Libevent::Global::initialize();
 
+  ::testing::Test::RecordProperty("TemporaryDirectory", TestEnvironment::temporaryDirectory());
```

Comment:
I'm not familiar with this. Can you put a short comment on what these 2 lines actually do?